homeassistant/const.py

```diff
@@ -228,6 +228,10 @@
 VOLUME_GALLONS = 'gal'  # type: str
 VOLUME_FLUID_OUNCE = 'fl. oz.'  # type: str
 
+# Bandwidth units
+BANDWIDTH_MEGABITS_SECONDS = 'Mbps'  # type: str
+BANDWIDTH_GIGABITS_SECONDS = 'Gbps'  # type: str
```

Comment:
Other sensors are already using the form `MB/s` and not the abbreviation `Mbps`. I think that we should stick with the existing format.